cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +66,26 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: gcr.io/google_containers/healthz-server:1.0
+        resources:
+          limits:
+            cpu: 100m
+            memory: 5Mi
```

Comment:
Yeah, totally makes sense especially given Tim's gc point. It's currently set to 20megs.